### ISSUE TITLE
googleDrive download- pre-trained model & VA_feature 

### ISSUE THREAD

**Issue by seohyeonShin** [GalaxyCong/HPMDubbing#9]
How have you been? 
I'm shamelessly creating issues. I started in February and 
now I'm using Emonet to create Feature 256 (this is all I have left)
Of course, I'm half-doubtful that this is made to be right..
What's curious about this part is that emotions are recognized for every cropped face image in the frame, but other features have npy for one image (basename), right? 
I don't know how VA_Feature(256dim) will be configured for successive scenes according to the entire basename.
**Frame/{basename}/00001.jpg or Cropped Face/{basename}/00001.jpg -> 1 VA Feature(256Dim)**
Do you make VA_features as an array for basename? Or, um, I want to know how you organized it.

so,, For this reason...
I need your pretrained model to get insight, could you please provide it?
Of course, I would really appreciate it if you could explain the VA Feature as well.

The provided google drive link needs your permission.
In the case of Baidu, I am a foreigner, so it is not easy to use.

**Comment by GalaxyCong** [comment]
I'm really sorry for the late response. 
Very good question！
In fact, each *.npy file stores a shape of (Video_length, dimention). In other words, instead of storing one *.npy file per frame, one *.npy file is stored for the entire video or according to the basename. Yes, an array of frames through concatenate 00001.jpg to the last 0000*.jpg！
About the link, and I will provide you with a new Google Drive link to convenient for you to download and use. 


- Extracted feature model: https://drive.google.com/file/d/19sKCQSMjxni60c0ANJxeUNKpw3zqJiTb/view?usp=drive_link
- Extracted features: https://drive.google.com/file/d/1Wf27AOHCLvkmIjycmE8eNi3BargxlYkS/view?usp=drive_link
(./example_V2C_framelevel/MovieAnimation/VA_feature)

**Comment by seohyeonShin** [comment]
I checked your history of changing readme two weeks ago. But I can't access the Google Drive link. May I ask you to allow me to access the link?
If you have to designate and allow access
My e-mail is

 ann4622.sin@gmail.com 

And I have an additional question. (I'm worried that you're tired of the continuous stream of questions, but..)
As you know, mfa doesn't include sentence unit division, you know?
So I took extra punctuation (because I didn't get a proper script from Chem youtube) and split the sentence and cut the data as a reference.
As a result, it seems to have failed.
Can you tell me in detail how you used mfa to divide it into sentence units?
If you look at the lab file created on movieanimation.py , it's a little different.
It's not even generated because it's numbered in units of sentences like your result screen.
(Is this part a secret know-how?)

**Comment by seohyeonShin** [comment]
Oh you've added the data I just noticed this, thank you so much.
wow,,I can't believe you sent me a feature... **I'll use your data dearly**

I've been trying it myself, and I couldn't find a place to get a comment to see if it was correct to split sentences(about chem)!
can you give me information about the sentence division of the chem data set!

**Comment by GalaxyCong** [comment]
Hi, my friend, 

mfa indeed doesn't include sentence unit division. 
In my opinion, it just aligns phoneme unit (or word level) with duration in speech. 

I roughly understand your doubts. Here is the original link of Chem dataset, you can download and check it:

https://mailstsinghuaeducn-my.sharepoint.com/:u:/g/personal/hu-cx21_mails_tsinghua_edu_cn/EWuyJYcDpLBBjgq2o94xay4B_wMtSx9Mlf1gdjQUO6jMWQ?e=TfhCLc

This link is from a wonderful job ([Neural Dubber](https://tsinghua-mars-lab.github.io/NeuralDubber/)).  They propose the Chem dataset with sentence-level clips according
to the start and end timestamp of each sentence in the transcripts (Maybe SRT subtitle files I think). 

In short, the sentences of Chem have been split before the data preprocess stage (i.e., MFA). 

It may solve your problem. If you still have questions, please continue to comment and I will continue to answer them 😊
